src/contracts/core/AllocationManager.sol

```diff
@@ -256,12 +262,12 @@ contract AllocationManager is
      * @param delay The allocation delay in seconds.
      */
     function _setAllocationDelay(address operator, uint32 delay) internal {
-        require(delay != 0, InvalidAllocationDelay());
```

Comment:
Nit: delete this error

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/842/commits/ba893331d545b5feb09573287e5f37b1fc63a27c